src/components/ToggleButton/ToggleButton.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import classnames from "classnames"
+import style from "./ToggleButton.module.css"
+
+export interface ToggleButtonProps  {
+  label?: string;
+  onChange: Function;
+  on?: boolean;
```

Comment:
I think we should make `on` required. Thoughts?

Reply:
Yes I agree